ui/desktop/src/components/ToolInvocations.tsx

```diff
@@ -137,23 +134,12 @@ function ToolResult({ result }: ToolResultProps) {
             )}
             {(isExpanded || !shouldMinimize) && (
               <>
-                {item.type === 'text' && item.text && (
+                {item.text && (
                   <MarkdownContent
                     content={item.text}
                     className="whitespace-pre-wrap p-2 max-w-full overflow-x-auto"
                   />
                 )}
-                {item.type === 'image' && item.data && item.mimeType && (
```

Comment:
Does this mean we drop images that could show up in `ToolResult`s?

Reply:
yeah accidental here will restore this path